screens/org-skillimpact-screen/OrgSkillImpactScreen.js

```diff
@@ -26,11 +35,11 @@ class OrgSkillImpactScreen extends Component {
   render() {
     //TODO view to be implemented
     return (
-      <View>
-
+      <View style={styles.container}>
+        <OrgSkilledImpactHeader/>
+        <OrgSkilledImpactBody/>
```

Comment:
Just curious (since I'm relatively unfamiliar w/ React Native best practices), but why did you decide to separate the header and body as different components? Is it to achieve more modularity or is there some functional reasoning behind it too?

Reply:
Yes 
1. is to keep it more organized since header and body will have different functionalities. The fetch functions are better off if they were written in these individual components.
2. it is too keep consistency with the entire code base since all pages are divided using individual components.